.github/workflows/triage.yml

```diff
@@ -146,10 +146,6 @@ jobs:
               path: Formula/(gettext|openssl@(3|1.1)|sqlite).rb
               keep_if_no_match: true
 
-            - label: CI-linux-wheezy
-              path: Formula/(zlib|binutils).rb
-              keep_if_no_match: true
```

Comment:
We should probably keep this for `zlib` until we also build `zlib` with `glibc@2.13`.

Reply:
I think @sjackman mentioned on Slack that `zlib` actually uses old enough `glibc` symbols that it does not need to be built with `glibc@2.13` - even when we build it with `glibc` 2.23 it will still work on older systems.

Reply:
Yeah we have built `zlib` with ubuntu 16.04 (by mistake) a few times over the last years in linuxbrew-core; and everything was fine. It just uses old enough `glibc` symbols. So I think we can safely remove this for now.